### PR TITLE
Added deployTo to build data

### PR DESCRIPTION
**Pull request by Jakob-em**
As descibed in #52

### CODE REVIEW

File: plugin.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ type (
 		Message string `json:"message"`
 		Status  string `json:"status"`
 		Link    string `json:"link"`
+		DeployTo string  `json:"deployTo"`
```
Comment (lafriks) [inline plugin.go]: Please format struct

Comment (Jakob-em) [reply plugin.go]: Done

File: go.mod [hunk 2]
```diff
@@ -10,3 +10,5 @@ require (
 	github.com/urfave/cli v1.20.0
 	gopkg.in/yaml.v2 v2.2.2 // indirect
 )
+
+go 1.13
```
Comment (lafriks) [inline go.mod]: Why this change? Anyway it should probably no the in this PR as it is not really related

Comment (Jakob-em) [reply go.mod]: Sorry I didn't want to stage this file. I will remove it
